### ISSUE TITLE
Run 9

### ISSUE THREAD

**Issue by jenkinsSAGrid** [SouthAfricaDigitalScience/libpng-deploy#19]
Ticket to track libpng-deploy/ARCH=x86_64,NAME=libpng,OS=u1404,SITE=generic,VERSION=1.6.20 build number 9

**Comment by brucellino** [comment]
running configure by hand on the machine doesn't give any errors. 

Just adding the CI module doesn't break the configure  and we get : 

```
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
```

**Comment by brucellino** [comment]
Adding zlib module does not trigger the failure and neither does specifying the right paths. This seems then to be a problem with the environment in which the job is run. I'm going to check the configuration.

**Comment by brucellino** [comment]
fixed in eb15b00.
Solution : spurious `\` in one of the configure lines.
